German/main/FileExplorer.apk/res/values-de/strings.xml

```diff
@@ -129,9 +129,9 @@
   <string name="downloading">Wird geladen</string>
   <string name="downloading_in_3g">Download über 3G...</string>
   <string name="enable_sd_card">SD-Karte einsetzen oder USB-Massenspeicher ausschalten.</string>
-  <string name="encrypt_error">"Dateien konnten nicht privat gemacht werden, nochmals versuchen"</string>
-  <string name="encrypt_prefix">PRIVAT</string>
-  <string name="encrypt_success">Alle Dateien sind nun privat</string>
+  <string name="encrypt_error">"Dateien können nicht Privat gesetzt werden. Versuchen Sie es erneut"</string>
+  <string name="encrypt_prefix">PRIVATE</string>
```

Comment:
Vom Deutschen in's Englische überschrieben. Fehlt in den "Korrektur"-Commits:
- https://github.com/muhamed89/MIUI/pull/311
- https://github.com/muhamed89/MIUI/pull/310